exposed-core/src/main/kotlin/org/jetbrains/exposed/sql/ColumnType.kt

```diff
@@ -266,6 +311,17 @@ abstract class StringColumnType(
         append('\'')
     }
 
+    override fun equals(other: Any?): Boolean {
+        if (this === other) return true
+        if (javaClass != other?.javaClass) return false
+
+        other as StringColumnType
+
+        if (collate != other.collate) return false
```

Comment:
Is nullability check missing here?

Reply:
Is safe:

if, `other` is null this comparative 

`if (javaClass != other?.javaClass) return false`

will be:

`if (javaClass != null) return false`

`javaClass` is always not null, then it will return `false` if `other` is null

Reply:
Sorry, i mean `nullable` on `StringColumnType`

Reply:
Its not necesary, because if other is null it will return false without try to cast it.

Reply:
`string("foo")` and `string("foo").nullable()` will be equal or not?

Reply:
Hahahahaha Sorry I've had not understood. I already fix it!.